openshift_pipelines/uninstalling-pipelines.adoc

```diff
@@ -0,0 +1,19 @@
+[id="uninstalling-pipelines"]
+= Uninstalling OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: uninstalling-pipelines
+
+toc::[]
+
+Uninstalling the {pipelines-title} Operator is a two step process. First, you delete the custom resources, which are added by default when you install the Operator. Second, you uninstall the {pipelines-title} Operator.
+
+Simply uninstalling the Operator will not remove the {pipelines-title} components created by default when the Operator is installed.
```

Comment:
Remove 'Simply' IBM style guide asks us to avoid this word because it is usually superfluous.

Reply:
done